code/modules/projectiles/projectile/magic/spellcard.dm

```diff
@@ -4,3 +4,60 @@
 	icon_state = "spellcard"
 	damage_type = BURN
 	damage = 2
+
+/obj/item/projectile/magic/spellcard/book
+	nodamage = FALSE
```

Comment:
Already false, no need to specify it again.

Reply:
No its not

Reply:
``/obj/item/projectile/magic
	name = "bolt of nothing"
	icon_state = "energy"
	damage = 0
	damage_type = OXY
	nodamage = 1
	armour_penetration = 100
	flag = "magic" ``
I dont want it to be TRUE I want that to be FALSE